templates/submit-with-summary.html

```diff
@@ -0,0 +1,23 @@
+{% extends 'layouts/_submit.html' %}
```

Comment:
This is slightly inconsistent when you look at calculatedsummary and hub which also extends submit. Following that pattern this should just be summary.

Reply:
Calculated summary has its own template now and doesn't share with submit.